cmk/gui/plugins/wato/check_parameters/checkpoint_connections.py

```diff
@@ -5,27 +5,32 @@
 # conditions defined in the file COPYING, which is part of this source code package.
 
 from cmk.gui.i18n import _
-from cmk.gui.valuespec import (
-    Integer,
-    Tuple,
-)
-
 from cmk.gui.plugins.wato import (
     CheckParameterRulespecWithoutItem,
     rulespec_registry,
     RulespecGroupCheckParametersApplications,
 )
+from cmk.gui.valuespec import Dictionary, Integer, Transform, Tuple
 
 
 def _parameter_valuespec_checkpoint_connections():
-    return Tuple(
-        help=_("This rule sets limits to the current number of connections through "
-               "a Checkpoint firewall."),
-        title=_("Maximum number of firewall connections"),
-        elements=[
-            Integer(title=_("Warning at"), default_value=40000),
-            Integer(title=_("Critical at"), default_value=50000),
-        ],
+    return Transform(
+        Dictionary(elements=[
+            (
+                "levels",
+                Tuple(help=_("This rule sets limits to the current number of connections through "
+                             "a Checkpoint firewall."),
+                      title=_("Maximum number of firewall connections"),
+                      elements=[
+                          Integer(title=_("Warning at"), minvalue=0.0),
```

Comment:
Please use an integer for minvalue (mypy also complains about this).

Reply:
Please keep the default values.

Reply:
will do, I actually got so many complaints from the tests (mostly referencing .cee modules which of course I don't have) that I missed the relevant bits last night..

Reply:
Thanks. We had some issues with the tests lately, but they should be fixed now (you need to rebase onto the latest master).